src/tag/tag.service.ts

```diff
@@ -5,7 +5,19 @@ import { DbService } from '../db/db.service';
 export class TagService {
   constructor(private readonly prisma: DbService) {}
 
-  async getAllTags() {
-    return await this.prisma.tag.findMany();
+  async searchTags(search: string) {
+    const exactMatch = await this.prisma.tag.findFirst({
+      where: { name: search },
+    });
+    const partialMatch = await this.prisma.tag.findMany({
+      where: { name: { contains: search } },
```

Comment:
it should starts with search, not include

Reply:
Done